index.css

```diff
@@ -488,21 +489,19 @@ html * {
   }
 
   .modal {
-    width: 100%;
-    height: 100%;
     background-color: rgba(255, 255, 255, 0.7);
     position: fixed;
     left: 0;
     top: 0;
-    z-index: 2;
+    right: 0;
   }
 
   .modal-body {
     position: relative;
     width: 90%;
     height: 90%;
     background-color: #fff;
-    margin: 3vw auto;
+    margin: 2vw auto;
     border-radius: 1rem;
     border: 1px solid #ebecf0;
   }
```

Comment:
- [ ] Please kindly reduce the height of this modal, as the previous reviewers highlighted and you can also add overflow-y as well so that all your content will be accessible.. 👍 
Please do this for both mobile and desktop version. Be encouraged, you can do it. 👍 
```
Height: 600px;
overflow-y: scroll;
overflow-x: hidden;
```

Reply:
@OyePriscilla  I think the template must be more specific because there not scroll in the design.

Reply:
@anderCM yeah!, but the text you added to the pop-up is wordy sort of. There are couple of ways to work around it anyway, I only gave suggestion. Thanks.

Reply:
@OyePriscilla I appreciate it, thanks.